README.md

```diff
@@ -117,7 +117,6 @@ cargo tauri icon
 ```
 
 ### Troubleshooting
-
```

Comment:
Why was this removed? Applying the standard formatter puts the blank line back.

Reply:
fixed